styles.css

```diff
@@ -0,0 +1,397 @@
+}
+
+.logo-grid-leftbar:hover {
+  cursor: pointer;
+}
+
+.options-nav {
+  grid-area: options-nav;
+  /* border: 1px dashed black; */
+
+  /* Make a grid within options-nav for the options */
+
+  display: grid;
+  grid-template-rows: repeat (80px 8);
+  grid-template-areas:
+    "home"
+    "explore"
+    "notifications"
+    "messages"
+    "bookmarks"
+    "lists"
+    "profile"
+    "more";
+}
+
+/* Options on left-bar */
+.home {
+  grid-area: home;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.home:hover {
+  cursor: pointer;
+}
+
+.explore {
+  grid-area: explore;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.explore:hover {
+  cursor: pointer;
+}
+
+.notifications {
+  grid-area: notifications;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.notifications:hover {
+  cursor: pointer;
+}
+
+.messages {
+  grid-area: messages;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.messages:hover {
+  cursor: pointer;
+}
+
+.bookmarks {
+  grid-area: bookmarks;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.bookmarks:hover {
+  cursor: pointer;
+}
+
+.lists {
+  grid-area: lists;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.lists:hover {
+  cursor: pointer;
+}
+
+.profile {
+  grid-area: profile;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.profile:hover {
+  cursor: pointer;
+}
+
+.more {
+  grid-area: more;
+  text-align: left;
+  margin-left: 70px;
+}
+
+.more:hover {
+  cursor: pointer;
+}
+
+/* Tweet button on left-bar */
+.tweet-grid {
+  grid-area: tweet-grid;
+  justify-self: center;
+}
+
+.tweet-button1 {
+  border-radius: 600px;
+  width: 200px;
+  height: 45px;
+  transform: translateY(100%);
+  background-color: #00acee;
+  color: white;
+  font-size: 15px;
+
+  border: none;
+}
+
+.tweet-button1:hover {
+  cursor: pointer;
+}
+/* Main */
+.main {
+  grid-area: main;
+  background-color: #f5f8fa;
+
+  /* Make a display grid within the main. The main will have three rows - home-page, posting section and feed section  */
+  display: grid;
+  grid-template-rows: 50px 130px 620px;
+  grid-template-areas:
+    "home-page"
+    "posting-section"
+    "feed";
+}
+
+.home-page {
+  grid-area: home-page;
+  margin-left: 80px;
+}
+
+.home-page:hover {
+  cursor: pointer;
+}
+
+.posting-section {
+  grid-area: posting-section;
+  /* border: 1px dashed black; */
+}
+
+.profile-picture {
+  width: 50px;
+  position: absolute;
+  bottom: 875px;
+  right: 1250px;
+  left: 350px;
+}
+
+.whats-happening {
+  margin-left: 140px;
+  margin-top: 10px;
+  font-size: 20px;
+  color: grey;
+
+  /* Making grid inside 'what's happening a grid to fit the icons in */
+  display: grid;
+  grid-template-columns: 300px;
+  grid-template-areas: uploading;
+  /* border: 1px solid black; */
+}
+
+.uploading {
+  grid-area: uploading;
+  /* border: 1px solid black; */
+}
+.media-upload {
+  width: 30px;
+  margin-left: 143px;
+}
+
+.media-upload:hover {
+  cursor: pointer;
+}
+
+.gif-upload {
+  width: 25px;
+  position: relative;
```

Comment:
Used relative positioning for all media icons. Is there a better way of positioning icon?